bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -196,7 +196,12 @@ public String getId() {
     @JsonIgnore
     @Nullable
     public String getRedirectUrl() {
-        return getContextPath() + GuideUtils.getRedirectUrl(redirect, getPath());
+        String redirectURL = redirect;
+        // Only do this for relative URLs, that is, ones hosted in AEM itself
+        if (StringUtils.isNoneEmpty(redirect) && redirect.startsWith("/")) {
```

Comment:
Since `redirect` is a string can't we use `StringUtils.isNotEmpty(redirect)` ?

Reply:
I was typing not, IDE pushed the top most suggestion.. :)